java/java.j2seplatform/src/org/netbeans/modules/java/j2seplatform/platformdefinition/DefaultPlatformImpl.java

```diff
@@ -84,6 +102,12 @@ static JavaPlatform create(Map<String,String> properties, List<URL> sources, Lis
         return new DefaultPlatformImpl(installFolders, properties, systemProperties, sources, javadoc);
     }
 
+    private static boolean isValidJDKHome(Path javaHome) {
+        return Files.exists(javaHome.resolve(Path.of("bin", "java")))
+            && Files.exists(javaHome.resolve(Path.of("bin", "javac")));
+//            && Files.exists(javaHome.resolve(Path.of("jmods", "java.base.mod")));   // TODO: uncomment once everything migrated to 11+
```

Comment:
Did you test this? Because that's probably still returning `true`.  The IDE wouldn't launch otherwise.  The problem is that those files are symlinks.  Are there any circumstances that `java.home` will not be correct?  I would either default to that, or at least default to that if `<jdk.home>/bin/java` is a symlink.

Reply:
I tested this with the jmods line uncommented, but then I realized that this is probably used by VSCode too.

So you are saying the `jdk.home` property has no use at this point in the lifecycle anymore, I should simply change it to `java.home`, do no validation and therefore also no notifications since it can't ever be wrong at this point?

Reply:
Well, I was going to put in a PR along the lines of the second option. Think that's probably safer? Using `jdk.home` was presumably added for some reason.

Reply:
ok changed, this makes things easier. Although having jdk.home set to /usr seems to be a fairly specific misconfiguration scenario. I am sure there are more creative ways how to have an invalid jdk.home set? I guess we will see in future :)

Reply:
@neilcsmith-net I should have let you do the PR. It probably took you as much time answering my questions as if you would have done it right away. Its just that I had it already started together with the NPE PR and split it later.